src/lp_data/HighsInfo.h

```diff
@@ -279,6 +281,18 @@ class HighsInfo : public HighsInfoStruct {
         new InfoRecordDouble("primal_dual_integral", "Primal-dual integral",
                              advanced, &primal_dual_integral, 0);
     records.push_back(record_double);
+
+    record_int =
+        new InfoRecordInt("max_cr_iteration_count1",
```

Comment:
```suggestion
        new InfoRecordInt("max_cr_diag_iteration_count",
```

Do we really want to name this `count1` and `count2`?

Reply:
Count1 and count2 are consist with Lukas' coding of IPX. These values will disappear when I've finished working on the branch. 

Yes, should have been a draft PR.